stations.csv

```diff
@@ -24920,7 +24920,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;is_city;country;
 27046;Comunion;comunion;;;42.7171582;-2.9668538;;t;ES;f;Europe/Madrid;f;;;f;;f;;f;eztgh2;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
 27047;Congosto;congosto;;;42.61755;-6.52048;;t;ES;f;Europe/Madrid;f;;;f;;f;;f;eze67w;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
 27048;Cortegana;cortegana;;;37.91116;-6.82045;;t;ES;f;Europe/Madrid;f;;;f;;f;;f;eyepsv;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
-27049;A Coruña;a-coruna;;;43.37135;-8.396;;t;ES;f;Europe/Madrid;f;;;f;;f;;f;ezdn8z;t;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;La Corogne;;;;;;ア・コルーニャ;라코루냐;;Corunha;;;;拉科魯尼亞
+27049;A Coruña;a-coruna;;;43.37135;-8.396;;t;ES;f;Europe/Madrid;t;;;f;;f;;f;ezdn8z;t;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
```

Comment:
Hey @mckbrmn, it looks like you merged https://github.com/trainline-eu/stations/commit/ca333904137a3b97b91409a9f8924b67e1189a24#diff-4f9a38aff8003d38921215c84fec67b3R234.

Is there a reason we should not relax that requirement? It's unfortunate to *lose* translations as a result of that…